appservice/src/intTest/groovy/nu/yona/server/DeviceManagementTest.groovy

```diff
@@ -514,6 +514,34 @@ class DeviceManagementTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
+	def 'Richard deletes a devices on which activities were done'()
```

Comment:
Why do we want to retain activities from the deleted device? If they have any functional use, should this test also not assert that the activities are still there?

Reply:
Asserting the activities is a good one. I'll add that. 

I wanted to retain the activities because they really happened. Otherwise, it would be possible to wipe out a significant part of past activities by dropping a device, which doesn't sound fair to buddies.